### ISSUE TITLE
Simulator and Device Testing Fails - Sends Request to Google, Not My App

### ISSUE THREAD

**Issue by nathanjliu** [actions-on-google/actions-on-google-nodejs#202]
This is an issue that began yesterday, for me and my team.

When I invoke my draft agent in google assistant and then send a query, the query gets sent to the normal Google Assistant to do a web search, instead of my agent. My agent then closes.

For example:

- "Talk to Agent" - invokes the agent and welcome intent
- "What's the temperature" - closes the agent and searches Google for the temperature.

This also happens with commands that Google does not have its own response for - I get a google search outside my agent.

**Comment by cwdcwd** [comment]
I've had that happen to me too. 
This is also similar to what I've experienced on ticket #189
It's like the session just dies but I'm stumped as to what to do next

**Comment by cwdcwd** [comment]
thanks @nathanjliu !

**Comment by cwdcwd** [comment]
@nathanjliu I opened a ticket as well but have not heard back. Any luck on your side?

**Comment by nathanjliu** [comment]
@cwdcwd I followed up today and they replied to say they're still investigating. I actually tried it today and it was working, so maybe they've changed something on the backend. 

Still not sure it's entirely fixed until I hear back from them so not closing this issue yet.

**Comment by cwdcwd** [comment]
thanks for the update @nathanjliu 
I still haven't heard from them.

**Comment by Canain** [comment]
This might have been an issue where the interaction with the action has timed out without the user knowing.

Can you still reproduce now?

**Comment by nathanjliu** [comment]
@Canain Yes, it is still reproducible. 

That doesn't seem to be the issue, I get an immediate response to my requests, just responses from Google Assistant not my agent. The query never reaches my dialogflow agent.

**Comment by nathanjliu** [comment]
This has been an ongoing issue for nearly three months now, and official support hasn't been much help. We're desperate for a solution here? Is there another channel where we can get some help please! @Canain

**Comment by Canain** [comment]
@nathanjliu Are you ensuring the conversation is continued?

Can you send screenshots of what you are observing?

**Comment by nathanjliu** [comment]
@Canain The conversation ends after I send a message. The issue is my message never reaches my agent - Google Assistant seems to 'forget' that I've invoked my own agent, and responds as if I haven't.

There's a long stack overflow chat I had with a 'Google Developer Expert' with some extra information. https://chat.stackoverflow.com/transcript/176417/2018/7/20

Below is a screenshot. I am executing what is essentially a debug command that should invoke an intent, and does invoke an intent when I send it in the Dialogflow console and other integrations. It should **not** execute a phone call, but set a context and return a text response. But Google Assistant tries to execute a phone call. 

<img width="1429" alt="screenshot 2018-10-15 at 09 34 33" src="https://user-images.githubusercontent.com/39554554/46939180-8d8a7f00-d05d-11e8-87f0-2fee606b7446.png">
 
Similarly, if I say something that I do not have an intent for, instead of returning my fallback intent, it tries to do a Google search, as if I have asked the question to the Google Assistant, not my agent. Below I ask for a capital city - a question irrelevant to my agent and that should return a fallback response, but it never reaches my agent, it goes to Google Assistant.

<img width="1426" alt="screenshot 2018-10-15 at 09 36 57" src="https://user-images.githubusercontent.com/39554554/46939307-e0fccd00-d05d-11e8-8073-3bfa2b633bf2.png">

The incorrect responses are the same on Google Home Mini and the Assistant app on iOS. The 'Request' and 'Response' tabs in simulator are blank.

Thanks for your help.

**Comment by entertailion** [comment]
Please contact the support team so they can debug this: https://developers.google.com/actions/support/

**Comment by nathanjliu** [comment]
@entertailion I have, months ago. They have failed to help, for months I have been facing automated 'we're working on this.' As a company we're pretty desperate, and can't find any help anywhere :(

**Comment by entertailion** [comment]
Did you get any support ticket number or ID that I can use to track this down?

**Comment by nathanjliu** [comment]
@entertailion No, but I Christian Dale was the support agent assigned, and the subject was "Simulator and Device Testing Fails - Sends Request to Google, Not My App." I have asked for a ticket number and will let you know if you can't find it without!

Thanks very much :)
